requirements/pyinstaller-build.txt

```diff
@@ -1,3 +1,5 @@
 # Executable binary builder requirements
-setuptools==70.3.0
+
+backports.tarfile==1.2.0; python_version=="3.8"
```

Comment:
It seems like it is adding a new dependency condition, can we check if it is expected? I couldn't find it in the reproducible files as well.

Reply:
its part of the setuptools import, but due to something that was changed it doesn't successfully import on python 3.8 anymore https://github.com/pypa/setuptools/issues/4508. That issue is closed so it might be fixed now in the newest release

Reply:
I can close this PR for dependabot to generate a new setuptools version bump PR.